diff --git a/src/tool/omniidl/python2/scripts/omniidl.in b/src/tool/omniidl/python2/scripts/omniidl.in
index f94515d..1c8602d 100644
--- a/src/tool/omniidl/python2/scripts/omniidl.in
+++ b/src/tool/omniidl/python2/scripts/omniidl.in
@@ -1,4 +1,4 @@
-#!@PYTHON@
+#!/usr/bin/env python
 # -*- python -*-
 #                           Package   : omniidl
 # omniidl.in                Created on: 1999/10/29
diff --git a/src/tool/omniidl/python3/scripts/omniidl.in b/src/tool/omniidl/python3/scripts/omniidl.in
index 5cbac89..d45e138 100644
--- a/src/tool/omniidl/python3/scripts/omniidl.in
+++ b/src/tool/omniidl/python3/scripts/omniidl.in
@@ -1,4 +1,4 @@
-#!@PYTHON@
+#!/usr/bin/env python
 # -*- python -*-
 #                           Package   : omniidl
 # omniidl.in                Created on: 1999/10/29
