diff --git a/src/lib/omniORB/dir.mk b/src/lib/omniORB/dir.mk
index caa5977..fdf5b77 100644
--- a/src/lib/omniORB/dir.mk
+++ b/src/lib/omniORB/dir.mk
@@ -1,7 +1,7 @@
 # dir.mk for omniORB.
 #
 
-PYSUBDIR = $(shell $(PYTHON) -c 'import sys; sys.stdout.write(sys.version[0] == "3" and "python3" or "python2")')
+PYSUBDIR = python3
 
 ifndef CrossCompiling
 SUBDIRS = $(PYSUBDIR)
diff --git a/src/tool/omniidl/dir.mk b/src/tool/omniidl/dir.mk
index 8ce53fb..5b39d90 100644
--- a/src/tool/omniidl/dir.mk
+++ b/src/tool/omniidl/dir.mk
@@ -5,7 +5,7 @@ export::
 	@$(NoPythonError)
 endif
 
-PYSUBDIR = $(shell $(PYTHON) -c 'import sys; sys.stdout.write(sys.version[0] == "3" and "python3" or "python2")')
+PYSUBDIR = python3
 
 SUBDIRS = cxx $(PYSUBDIR)
 
