Index: configure
===================================================================
--- configure	(revision 6648)
+++ configure	(revision 6649)
@@ -8576,6 +8576,7 @@
   alpha*)   proc_name="AlphaProcessor";   proc_def="__alpha__";;
   m68k-*)   proc_name="m68kProcessor";    proc_def="__m68k__";;
   mips*)    proc_name="IndigoProcessor";  proc_def="__mips__";;
+  arm64-*)  proc_name="Arm64Processor";   proc_def="__arm64__";;
   arm-*)    proc_name="ArmProcessor";     proc_def="__arm__";;
   s390-*)   proc_name="s390Processor";    proc_def="__s390__";;
   ia64-*)   proc_name="ia64Processor";    proc_def="__ia64__";;
Index: configure.ac
===================================================================
--- configure.ac	(revision 6648)
+++ configure.ac	(revision 6649)
@@ -249,6 +249,7 @@
   alpha*)   proc_name="AlphaProcessor";   proc_def="__alpha__";;
   m68k-*)   proc_name="m68kProcessor";    proc_def="__m68k__";;
   mips*)    proc_name="IndigoProcessor";  proc_def="__mips__";;
+  arm64-*)  proc_name="Arm64Processor";   proc_def="__arm64__";;
   arm-*)    proc_name="ArmProcessor";     proc_def="__arm__";;
   s390-*)   proc_name="s390Processor";    proc_def="__s390__";;
   ia64-*)   proc_name="ia64Processor";    proc_def="__ia64__";;
Index: include/omniORB4/CORBA_sysdep.h
===================================================================
--- include/omniORB4/CORBA_sysdep.h	(revision 6648)
+++ include/omniORB4/CORBA_sysdep.h	(revision 6649)
@@ -92,7 +92,7 @@
 // __VFP_FP__ means that the floating point format in use is that of the ARM 
 // VFP unit, which is native-endian IEEE-754.
 #if defined(__arm__)
-#  if defined(__armv5teb__) || defined(__VFP_FP__) || defined(__aarch64__)
+#  if defined(__armv5teb__) || defined(__VFP_FP__) || defined(__aarch64__) || defined(__arm64__)
 #    define NO_OMNI_MIXED_ENDIAN_DOUBLE
 #  else
 #    define OMNI_MIXED_ENDIAN_DOUBLE
Index: include/omniconfig.h.in
===================================================================
--- include/omniconfig.h.in	(revision 6648)
+++ include/omniconfig.h.in	(revision 6649)
@@ -3,7 +3,7 @@
  * omniconfig.h.in            Created on: 2002/07/11
  *                            Author    : Duncan Grisby (dgrisby)
  *
- *    Copyright (C) 2002 Duncan Grisby
+ *    Copyright (C) 2002-2023 Duncan Grisby
  *
  *    This file is part of the omniORB library
  *
@@ -33,8 +33,41 @@
 
 #include <omniORB4/acconfig.h>
 
-#define @PLATFORM_DEFINE@ 1
-#define @PROCESSOR_DEFINE@ 1
+#if !defined(__linux__)          && \
+    !defined(__cygwin__)         && \
+    !defined(__sunos__)          && \
+    !defined(__osf1__)           && \
+    !defined(__hpux__)           && \
+    !defined(__nextstep__)       && \
+    !defined(__irix__)           && \
+    !defined(__aix__)            && \
+    !defined(__darwin__)         && \
+    !defined(__freebsd__)        && \
+    !defined(__FreeBSD_kernel__) && \
+    !defined(__netbsd__)         && \
+    !defined(__openbsd__)        && \
+    !defined(__osr5__)           && \
+    !defined(__hurd__)           && \
+    !defined(__win32__)
+# define @PLATFORM_DEFINE@ 1
+#endif
+
+#if !defined(__x86__)     && \
+    !defined(__x86_64__)  && \
+    !defined(__sparc__)   && \
+    !defined(__alpha__)   && \
+    !defined(__m68k__)    && \
+    !defined(__mips__)    && \
+    !defined(__arm__)     && \
+    !defined(__arm64__)   && \
+    !defined(__s390__)    && \
+    !defined(__ia64__)    && \
+    !defined(__hppa__)    && \
+    !defined(__powerpc__) && \
+    !defined(__riscv)
+# define @PROCESSOR_DEFINE@ 1
+#endif
+
 #define __OSVERSION__ @OSVERSION@
 
 #undef PACKAGE_BUGREPORT
