Index: mk/beforeauto.mk.in
===================================================================
--- mk/beforeauto.mk.in	(revision 6653)
+++ mk/beforeauto.mk.in	(revision 6654)
@@ -856,7 +856,6 @@
 
 ###################
 ifdef Linux
-IMPORT_CPPFLAGS += -D__linux__
 OMNITHREAD_POSIX_CPPFLAGS = -DNoNanoSleep -DPthreadDraftVersion=10
 OMNITHREAD_CPPFLAGS = -D_REENTRANT
 OMNITHREAD_LIB += -lpthread
@@ -878,7 +877,6 @@
 
 ###################
 ifdef SunOS
-IMPORT_CPPFLAGS += -D__sunos__
 OMNITHREAD_POSIX_CPPFLAGS = -DPthreadDraftVersion=10
 OMNITHREAD_CPPFLAGS = -DUsePthread -D_REENTRANT $(CXXMTFLAG)
 SOCKET_LIB = -lsocket -lnsl
@@ -909,8 +907,6 @@
 
 ###################
 ifdef OSF1
-IMPORT_CPPFLAGS          += -D__osf1__
-
 ifeq (@OSVERSION@,3)
 OMNITHREAD_POSIX_CPPFLAGS = -DPthreadDraftVersion=4 -DNoNanoSleep
 else
@@ -956,7 +952,6 @@
 
 ###################
 ifdef HPUX
-IMPORT_CPPFLAGS += -D__hpux__
 INSTLIBFLAGS     = -m 0755
 
 ifeq (@OSVERSION@,10)
@@ -1017,7 +1012,6 @@
 
 ###################
 ifdef NextStep
-IMPORT_CPPFLAGS += -D__nextstep__
 
 ThreadSystem = Mach
 OMNITHREAD_CPPFLAGS = -D_REENTRANT
@@ -1029,8 +1023,6 @@
 
 ###################
 ifdef IRIX
-IMPORT_CPPFLAGS += -D__irix__
-
 OMNITHREAD_POSIX_CPPFLAGS = -DPthreadDraftVersion=10 \
 			    -DPthreadSupportThreadPriority
 OMNITHREAD_LIB += -lpthread
@@ -1053,8 +1045,6 @@
 ifdef AIX
 ifndef Compiler_GCC		# Assuming XlC compiler
 
-IMPORT_CPPFLAGS += -D__aix__
-
 CMAKEDEPEND     += -D_AIX
 CXXMAKEDEPEND   += -D_AIX
 CDEBUGFLAGS      =
@@ -1080,7 +1070,6 @@
 
 ifdef Compiler_GCC
 
-IMPORT_CPPFLAGS += -D__aix__
 CMAKEDEPEND     += -D_AIX
 CXXMAKEDEPEND   += -D_AIX
 CXXLINKOPTIONS  += -Wl,-brtl -Wl,-blibpath:/lib:/usr/lib:$(prefix)/lib
@@ -1120,8 +1109,6 @@
 
 ###################
 ifdef Darwin
-IMPORT_CPPFLAGS += -D__darwin__
-
 OMNITHREAD_POSIX_CPPFLAGS = -DPthreadDraftVersion=10 \
                             -DPthreadSupportThreadPriority -DNoNanoSleep
 
@@ -1160,8 +1147,6 @@
 
 ###################
 ifdef FreeBSD
-IMPORT_CPPFLAGS += -D__freebsd__
-
 OMNITHREAD_CPPFLAGS = -D_REENTRANT -D_THREAD_SAFE
 OMNITHREAD_POSIX_CPPFLAGS = -DUsePthread -DPthreadDraftVersion=10
 OMNITHREAD_LIB += -pthread
@@ -1169,8 +1154,6 @@
 
 ###################
 ifdef NetBSD
-IMPORT_CPPFLAGS += -D__netbsd__
-
 OMNITHREAD_CPPFLAGS = -D_REENTRANT
 OMNITHREAD_POSIX_CPPFLAGS = -DUsePthread -DPthreadDraftVersion=10
 OMNITHREAD_LIB += -pthread
@@ -1178,8 +1161,6 @@
 
 ###################
 ifdef OpenBSD
-IMPORT_CPPFLAGS += -D__openbsd__
-
 OMNITHREAD_CPPFLAGS = -D_REENTRANT -D_THREAD_SAFE
 OMNITHREAD_POSIX_CPPFLAGS = -DUsePthread -DPthreadDraftVersion=10
 OMNITHREAD_LIB += -pthread
@@ -1187,8 +1168,6 @@
 
 ###################
 ifdef OSR5
-IMPORT_CPPFLAGS += -D__osr5__
-
 COPTIONS = -fpcc-struct-return
 
 OMNITHREAD_POSIX_CPPFLAGS = -DPthreadDraftVersion=6 \
@@ -1199,7 +1178,6 @@
 ifdef Cygwin
 MKDIRHIER = mkdir -p
 CXXLINKOPTIONS += -Wl,--enable-auto-import
-IMPORT_CPPFLAGS += -D__cygwin__
 SHAREDLIB_CPPFLAGS =
 OMNITHREAD_POSIX_CPPFLAGS = -DNoNanoSleep -DPthreadDraftVersion=10
 OMNITHREAD_CPPFLAGS = -D_REENTRANT
@@ -1258,56 +1236,6 @@
 
 ###########################################################################
 #
-# Processor
-#
-
-ifdef x86Processor
-IMPORT_CPPFLAGS += -D__x86__
-endif
-
-ifdef x8664Processor
-IMPORT_CPPFLAGS += -D__x86_64__
-endif
-
-ifdef SparcProcessor
-IMPORT_CPPFLAGS += -D__sparc__
-endif
-
-ifdef AlphaProcessor
-IMPORT_CPPFLAGS += -D__alpha__
-endif
-
-ifdef m68kProcessor
-IMPORT_CPPFLAGS += -D__m68k__
-endif
-
-ifdef IndigoProcessor
-IMPORT_CPPFLAGS += -D__mips__
-endif
-
-ifdef ArmProcessor
-IMPORT_CPPFLAGS += -D__arm__
-endif
-
-ifdef s390Processor
-IMPORT_CPPFLAGS += -D__s390__
-endif
-
-ifdef ia64Processor
-IMPORT_CPPFLAGS += -D__ia64__
-endif
-
-ifdef HppaProcessor
-IMPORT_CPPFLAGS += -D__hppa__
-endif
-
-ifdef PowerPCProcessor
-IMPORT_CPPFLAGS += -D__powerpc__
-endif
-
-
-###########################################################################
-#
 # Final things
 #
